.github/workflows/release.yml

```
@@ -161,6 +161,39 @@ jobs:
           name: ${{ runner.os }}-wheels
           path: wheelhouse
 
+  macos-arm-release-wheel:
+    name: Build release arm wheels for macOS
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Actually, I don't think we need to build it against 3.6 and 3.7, since only [Python 3.8 and 3.9 are supported by `tensorflow-macos`](https://pypi.org/project/tensorflow-macos).
```suggestion
        python-version: [3.8, 3.9]
```